src/bp/ui-studio/src/web/views/OneFlow/diagram/RouterForm/style.scss

```diff
@@ -0,0 +1,25 @@
+.formHeader {
```

Comment:
Unless you change values in this file, can you import the css of another form ? Will be easier to refactor !